### DISCUSSION TITLE
When using "change tab(s) to workspace" to send tabs to another workspace it should not switch the workspace

### DISCUSSION THREAD

**Discussion by badjuice** [zen-browser/desktop#1058] [Ideas]
### What happened?

When I right click tabs and select "change tab(s) to workspace" to send them to another workspace it should stay in the current workspace instead of switching the workspace.

If that's supposed to be the intented behavior then can it be please made into an option?

Thanks in advance :)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.28

### Severity impact

Low

### What platform are you seeing the problem on?

Windows

**Comment by alpiua** [comment]
Not so many frustrating things in Zen, but this behavior is one of them.
Totally for the option to stay on the current workspace while sending tab to another.

**Comment by emenel** [comment]
agreed. i really wish there was a "send to workspace" instead of only "switch"

**Comment by davidbelius** [comment]
I would also like to have a setting that switches the behavior in this way.
